Configuration/TCA/Overrides/pages.php

```diff
@@ -72,6 +75,6 @@
         'pages',
         '--palette--;LLL:EXT:newspage/Resources/Private/Language/locallang_be.xlf:news;tx_newspage,',
         $dokType,
-        'after:title'
+        'after:title',
```

Comment:
This comma shouldn't be here.
I don't like trailing commas, especially on function calls that have a set amount of parameters that will not change.

Additionally this breaks with PHP < 7.3. This is a conflict with TYPO3 10 PHP requirements (>= 7.2, <= 7.4) which I do not want to introduce.